gubernator/view_build.py

```diff
@@ -112,6 +112,39 @@ def get_running_build_log(job, build, prow_url):
     return None, None
 
 
+def normalize_metadata(started_future, finished_future):
+    """
+    Munge and normalize the output of loading started
+    and finished.json files from a GCS bucket.
+
+    :param started_future: future from gcs_async.read()
+    :param finished_future: future from gcs_async.read()
+    :return: started, finished dictionaries
+    """
+    started = started_future.get_result()
+    finished = finished_future.get_result()
+    if finished and not started:
+        started = 'null'
```

Comment:
this still won't work for view_pr.py

Reply:
Why?

Reply:
(and how do we write a test to expose that)?